packages/quantic/README.md

```diff
@@ -60,23 +60,29 @@ Example components are available as a Salesforce community (Digital Experience)
     sfdx force:community:create -u LWC -n "Quantic Examples" -p examples -t "Build Your Own"
     ```
 
-2. To deploy the community in the scratch org, run:
+2. To deploy the Quantic and examples components, run:
```

Comment:
```suggestion
2. To deploy Quantic and the example components, run:
```

Reply:
Unless you meant: `To deploy the Quantic example components, run:`

Reply:
Your first suggestion is perfect! Thanks